script

```diff
@@ -1042,7 +1110,7 @@ only_send_error_notifications="0"
 
 
   # create timestamp variable for rolling backups.
-  timestamp="$(date '+%Y%m%d_%H%M')""_"
+  timestamp="$(date '+%Y%m%d_%H%M%S')""_"
```

Comment:
I prefer to leave the seconds off of the backup timestamps. It makes the file names unnecessarily longer since knowing exactly which second you started a backup (something that takes several minutes to several hours) is rarely, if ever, useful information.

Reply:
Yeah, I can revert this one.  In the real world the seconds don't matter at all, but in my testing it was throwing errors if I tried to do multiple backups within the same minute. ;)

Reply:
This has been reverted